bin/hermit

```diff
@@ -15,14 +17,27 @@ if [ -z "${HERMIT_STATE_DIR}" ]; then
   esac
 fi
 
-export HERMIT_DIST_URL="${HERMIT_DIST_URL:-https://github.com/cashapp/hermit/releases/download/stable}"
+export HERMIT_DIST_URL="${HERMIT_DIST_URL:-https://github.com/cashapp/hermit/releases/download/canary}"
 HERMIT_CHANNEL="$(basename "${HERMIT_DIST_URL}")"
 export HERMIT_CHANNEL
 export HERMIT_EXE=${HERMIT_EXE:-${HERMIT_STATE_DIR}/pkg/hermit@${HERMIT_CHANNEL}/hermit}
 
 if [ ! -x "${HERMIT_EXE}" ]; then
   echo "Bootstrapping ${HERMIT_EXE} from ${HERMIT_DIST_URL}" 1>&2
-  curl -fsSL "${HERMIT_DIST_URL}/install.sh" | /bin/bash 1>&2
+  INSTALL_SCRIPT="$(mktemp)"
```

Comment:
It's _really_ not great to have too much shell in `bin/hermit`, because it slows down every invocation of `hermit`. I think this would be fine to pull out into a separate script though, perhaps `bin/download-hermit` or something? Also can we we make this a separate PR?

Reply:
I didn’t edit this. This is from `hermit init`

Reply:
Ah man 🤦

Reply:
Weird that I don't see that in any of my recent environments...so strange.